client/configuration_variable_test.go

```diff
@@ -187,9 +190,10 @@ var _ = Describe("Configuration Variable", func() {
 	})
 
 	Describe("ConfigurationVariableDelete", func() {
+
 		BeforeEach(func() {
 			httpCall = mockHttpClient.EXPECT().Delete("configuration/"+mockConfigurationVariable.Id, nil)
-			apiClient.ConfigurationVariableDelete(mockConfigurationVariable.Id)
+			_ = apiClient.ConfigurationVariableDelete(mockConfigurationVariable.Id)
```

Comment:
why these changes?

Reply:
using a new linter - fixes a linting issue.
It tells the linter, I know that this can return an error - but I want to ignore it.